new/new.go

```diff
@@ -13,13 +13,13 @@ import (
 )
 
 func MustNew(cfg *Config) {
-	if err := New(cfg.TemplName, cfg.Targets); err != nil {
+	if err := New(cfg.TemplName, cfg.Targets, cfg.KeepTempVals); err != nil {
 		fmt.Fprintf(os.Stderr, "error: %v\n", err)
 		os.Exit(1)
 	}
 }
 
-func New(templName string, targets []string) error {
+func New(templName string, targets []string, keepVals bool) error {
```

Comment:
refactor to just pass the config object to this function, instead of having so many params